instructions/index.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Vladimir</title>
+</head>
+<body>
+
+</body>
+</html>
```

Comment:
This should be added in the top level folder/directory, not in the instructions sub-directory.

Reply:
Extra credit for adding html rather than just your name as the lesson called for!